rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/websphere/ResolveWebSphereWsBindingXmlRuleProvider.java

```diff
@@ -27,18 +30,41 @@
     @Override
     public ConditionBuilder when()
     {
-        return Query.fromType(XmlFileModel.class).withProperty(FileModel.FILE_NAME, "ibm-webservicesclient-bnd.xmi")
-                    .withProperty(XmlFileModel.ROOT_TAG_NAME, "ClientBinding");
+        return Query.fromType(XmlFileModel.class).withProperty(FileModel.FILE_NAME, "ibm-webservices-bnd.xmi")
+                    .withProperty(XmlFileModel.ROOT_TAG_NAME, "WSBinding");
     }
 
     @Override
     public void perform(GraphRewrite event, EvaluationContext context, XmlFileModel payload)
     {
         ClassificationService classificationService = new ClassificationService(event.getGraphContext());
-        classificationService.attachClassification(event, context, payload, IssueCategoryRegistry.MANDATORY, "WebSphere WS Binding", "WebSphere Webservice Binding XML Descriptor.");
+        ClassificationModel classificationModel = classificationService.attachClassification(event, context, payload, IssueCategoryRegistry.MANDATORY,
+                    "WebSphere WS Binding",
+                    "WebSphere Webservice Binding XML Deployment Descriptor.  \n"
+                                + "This deployment descriptor extension is IBM-specific and it needs to be migrated to JBossWS.  \n"
```

Comment:
I wrote it with the dash just because i found it with the dash in https://github.com/mrizzi/windup-rulesets/blob/master/rules-reviewed/eap6/websphere/websphere-xml.windup.xml#L153 but it's not a problem, i remove all the dashes ;)